perception/lidar_apollo_instance_segmentation/src/debugger.cpp

```diff
@@ -34,12 +34,12 @@ void Debugger::publishColoredPointCloud(
 {
   using autoware_perception_msgs::msg::ObjectClassification;
   pcl::PointCloud<pcl::PointXYZRGB> colored_pointcloud;
-  for (size_t i = 0; i < input.feature_objects.size(); i++) {
+  for (const auto & feature_object : input.feature_objects) {
```

Comment:
@kobayu858 
Consider using range-based for loops

Reply:
@veqcc 
What do you mean by what I think is already a range-based for loop?

Reply:
Sorry my comment was pointing to wrong line number.
I indicated the correct one in another comment.